### DISCUSSION TITLE
Bootstrap styles are not being overridden ONLY in production build

### DISCUSSION THREAD

**Discussion by saadbinsaeed** [vercel/next.js#21516] [Help]
Hi I am using next js version `10.0.4`. I just copied `bootstrap.min.css` in my project to use the bootstrap `v4.3.1` css. 

I have another css file `style.css` where I override some styles of bootstrap css. 

Every thing works perfect when I am in `development` mode using `npm run dev` 

But on production build and static export, my overridden styles don't work at all. All styles changes to default bootstrap styles. 

Anyone has any idea what could be the issue?

**Comment by salahCoone** [comment]
I also have the same problem

**Comment by icyJoseph** [comment]
@salahCoone @saadbinsaeed is this still happening with latest versions of Next.js?

The problem is most likely that the bootstrap stylesheet is being inserted last to the DOM. Why would this happen I am not sure. 

From what I read, you have something like:


```tsx
// pages/_app.js
import '../styles/bootstrap.min.css';
import '../styles/styles.css';
```

That should resolve into two link tags, first bootstrap, then the overrides, and by cascade the overrides should apply. Is that what you have? I've done that and see no issues. Tested with Next 12.1.4, and 10.0.4, both with `next dev` and `next build && next start`.

**Comment by mohadjillani** [comment]
I was facing the same issue. I fix it by changing the import of the bootstrap.
Instead of importing this: `import "bootstrap/dist/css/bootstrap.css";`
I import the Bootsrap CSS in the Head of `_app.js` like:

` <Head>
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
          crossorigin="anonymous"
        />
      </Head>`
      
![Screenshot from 2022-10-20 16-29-03](https://user-images.githubusercontent.com/39382468/196936806-41778e41-539e-4a18-afb6-cb72790afa38.png)

**Reply by BlueLovin** [reply]
wow, i hate this but it works!
